frontend/src/index.css

```
@@ -7456,11 +7456,1022 @@ textarea {
+  border-radius: 50%;
+}
+
+.native-library-track-list {
+  min-width: 0;
+  overflow-x: auto;
+  border-top: 1px solid var(--aurral-border);
+}
+
+.native-library-track {
+  --native-library-track-columns: 2rem 2rem 2.35rem minmax(12rem, 1.4fr) minmax(8rem, 1fr) minmax(8rem, 1fr) 4.5rem 1.7rem;
+  display: grid;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)